styles.css

```diff
@@ -340,6 +340,68 @@ main {
   color: var(--header-text-color);
 }
 
+.contact {
+  background-image: url('./assets/images/contact-from-bg.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  padding: 45px 24px;
+  background-position-x: right;
+  background-origin: border-box;
+  background-color: var(--primary-color);
+  border-top-left-radius: 15%;
+}
+
+.contact-me-info {
+  margin-top: 40px;
+  text-align: center;
```

Comment:
- [ ] Nice work you did there 👍 . However, based on project requirements, you're meant to use Flexbox to style this section. This is to test your skill using that technique, which is an industry best practice and gives you more flexibility 👍 

## Requirement:
![image](https://github.com/habitoz/portofolio/assets/50990925/7c84febd-8740-471e-b7ef-364b7d46add2)